### PR TITLE
complete CreateKey_napi 、Encrypt_napi、Decrypt_napi、GenerateDataKey_napi

### PR DESCRIPTION
**Pull request by Liang1XZhao**
The content includes nodejs and C + +

### CODE REVIEW

**Review by syan10 — APPROVED** [review]
LGTM. :)

**Review by syan10 — APPROVED** [review]
LGTM :)

File: ehsm_kms_service/test/test_aes.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
+import requests
```
Comment (syan10) [inline ehsm_kms_service/test/test_aes.py:1]: after the patch merged, you can submit another commit to do the following things:

1. rename it as test_kms_with_rest.py
2. following the same test flow(in the main.cpp) to refine the test cases:

test_aes_128(){
a. create cmk
b. encrypt a plaintext
c. decrypt the ciphertext and compare it's same with the plaintext.
}

test_generatedatakey(){
a. create cmk
b. generate a datakey with a keylen(e.g. keylen=16)
c. try to decrypt the cipher_datakey and compare it's same with plaint_datakey.
}

and later, you can add more test cases when you add the other interfaces(rsa_sign/rsa_verifiy, .....)

File: ehsm_kms_service/package.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "ehsm_kms_service",
+  "version": "1.0.0",
+  "description": "ehsm kms service",
+  "main": "ehsm_kms_server.js",
```
Comment (syan10) [inline ehsm_kms_service/package.json:5]: just a reminder, please add the license as "BSD" in next commit.

**Review by yang8621 — APPROVED** [review]

**Pull request merged**